usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -1869,9 +1870,17 @@ static void *ep11_load_host_lib()
         return lib_ep11;
     }
 
-    ep11_lib_name = EP11SHAREDLIB_V2;
+    ep11_lib_name = EP11SHAREDLIB_V3;
     lib_ep11 = dlopen(ep11_lib_name, RTLD_GLOBAL | RTLD_NOW);
 
+    if (lib_ep11 == NULL) {
```

Comment:
we have this if-statement now 3 times. for future releases it would make sense to have the libnames in an array and probe for lib availability from rightmost element downwards, so adding a new lib version would only mean adding it to the array and not copy the if-statement again.

Reply:
Yes I agree.